src/python_testing/TC_ACL_2_10.py

```diff
@@ -101,6 +101,9 @@ async def test_TC_ACL_2_10(self):
         self.step(1)
         self.th1 = self.default_controller
         self.discriminator = random.randint(0, 4095)
+        self.endpoint = self.get_endpoint()
+        extension_attr = Clusters.AccessControl.Attributes.Extension
+        ext_attr_enabled = await self.attribute_guard(endpoint=self.endpoint, attribute=extension_attr)
```

Comment:
attribute guard will mark the current step as skipped if it's false. 

You need to use the guard directly to get the test steps marked correctlyt.